ledger-core/v2/insolar/pulsestor/memory.go

```diff
@@ -3,39 +3,39 @@
 // This material is licensed under the Insolar License version 1.0,
 // available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
 
-package pulse
+package pulsestor
```

Comment:
hmm i wonder why it is pulsestor, not pulsestore? :)

Reply:
the name is ugly because I'd rather get rid of this package ;) but the change is already too big :(